ci-configs/packages-latest.json

```diff
@@ -2166,6 +2166,19 @@
         "sample*",
         "doc*"
       ]
+    },
+    {
+      "package_info": {
```

Comment:
I honestly expected this to come through as part of automation. I'll check your release.

Reply:
Ok I see. You do not need this part. Your package has already been onboarded into `packages-preview.json` as part of [this step](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=827902&view=logs&j=d3fa97dd-0d4d-54ee-ed7b-bd6dcd03e7ff&t=f086e903-00ed-5536-849f-b6711cb24ea3) of your release.